### ISSUE TITLE
Support :matches()

### ISSUE THREAD

**Issue by MoOx** [cssnext/cssnext#17]
```css
body :matches(h1, h2) {
  color: red;
}
```

should yields into

```css
body h1, body h2 {
  color: red;
}
```

poke @Eramdam

**Comment by MoOx** [comment]
We might want to port this https://github.com/michelle/rework-matches

**Comment by MoOx** [comment]
Done. I will release postcss-selector-matches and include it in cssnext tonight ir tomorrow :)

**Comment by magsout** [comment]
@MoOx good job (again?)

**Comment by MoOx** [comment]
BOOM https://github.com/postcss/postcss-selector-matches

**Comment by MoOx** [comment]
Closed by 0cf0c13bb20e1aa39c21aa5d733cabb1f7d9a11e

**Comment by MoOx** [comment]
Released as 1.4.0
